LaTeX/LaTeX Log.sublime-syntax

```diff
@@ -6,44 +6,81 @@ scope: text.log.latex
 first_line_match: "This is (pdf|pdfe)?TeXk?, Version "
 
 contexts:
+  eol-pop:
+    - match: '$\n?'
+      pop: 1
+
```

Comment:
The `main` context should always be the first one. Most generic/common prototype like contexts should be located at the end of the document.